exercises/simple-linked-list/src/example/java/SimpleLinkedList.java

```diff
@@ -0,0 +1,79 @@
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
+/**
+ *
+ * @author matt
+ */
+public class SimpleLinkedList {
+
+    class Element {
+
+        int value;
+        Element next;
+
+        Element(int value) {
+            this.value = value;
+        }
+    }
+
+    private Element head;
+    private int size = 0;
+
+    public SimpleLinkedList() {
+    }
+
+    public SimpleLinkedList(int[] values) {
+        for (int ii = values.length - 1; ii >= 0; ii--) {
```

Comment:
Yes, the Java `LinkedList` class provides `push` and `pop` methods that both operate on the head (LIFO), so that's what I was emulating, but if it should be different please let me know.